### PR TITLE
Remove demo shell protractor TreeView e2e

### PR DESCRIPTION
**Pull request by DenysVuika**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [x] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

- remove tree view page from demo shell
- remove e2e (protractor) testing demo shell (already covered by unit tests)

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: demo-shell/src/app/components/tree-view/tree-view-sample.component.html [hunk 1]
```diff
@@ -1,19 +0,0 @@
...
-<mat-form-field class="example-full-width">
-    <input matInput placeholder="Node Id"
-                [(ngModel)]="nodeIdSample"
-                (ngModelChange)="reset()">
-</mat-form-field>
-<span>
-    CLICKED NODE: {{clickedNodeName}}
-</span>
-
-<adf-tree-view-list *ngIf="!errorMessage; else errorOccurred"
```
[the remark this answers is not in the document]
Comment (DenysVuika) [reply demo-shell/src/app/components/tree-view/tree-view-sample.component.html:11]: Not used, probably needs to be removed together with other unused components in the next major upgrade

File: e2e/content-services/components/tree-view-component.e2e.ts [hunk 2]
```diff
@@ -1,139 +0,0 @@
...
-            nodeType: 'cm:folder'
-        });
-        await nodesApi.createNode(thirdTreeFolder.entry.id, {
-            name: nodeNames.document,
-            nodeType: 'cm:content'
-        });
-
-        await loginPage.login(acsUser.username, acsUser.password);
-
-        await navigationBarPage.clickTreeViewButton();
```
Comment (AleksanderSklorz) [inline e2e/content-services/components/tree-view-component.e2e.ts:71]: can this function be removed?

File: demo-shell/src/app/components/tree-view/tree-view-sample.component.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (AleksanderSklorz) [reply demo-shell/src/app/components/tree-view/tree-view-sample.component.html:11]: @DenysVuika could you please raise jira to don't forget if it won't be done as part of that PR?

**Review by AleksanderSklorz — APPROVED** [review]

File: demo-shell/src/app/components/tree-view/tree-view-sample.component.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (DenysVuika) [reply demo-shell/src/app/components/tree-view/tree-view-sample.component.html:11]: https://alfresco.atlassian.net/browse/ACS-6124

[the remark this answers is not in the document]
Comment (AleksanderSklorz) [reply demo-shell/src/app/components/tree-view/tree-view-sample.component.html:11]: thank you

**Pull request merged**